compiler/rustc_middle/src/mir/mono.rs

```diff
@@ -103,17 +102,13 @@ impl<'tcx> MonoItem<'tcx> {
                 // inlined function. If we're inlining into all CGUs then we'll
                 // be creating a local copy per CGU.
                 if generate_cgu_internal_copies {
```

Comment:
It wasn't immediately obvious to me that `generate_cpu_internal_copies` had any relation to whether or not we're compiling in debug mode. Could we make that clearer by changing the name of the variable or adding a comment?

`generate_cpu_internal_copies` seems to only be used in this branch of the `match`, it might make things a tad easier to read to define it just before the condition?

Reply:
Good idea!